ebos/eclcpgridvanguard.hh

```diff
@@ -83,8 +83,11 @@ private:
 
 public:
     EclCpGridVanguard(Simulator& simulator)
-        : EclBaseVanguard<TypeTag>(simulator)
+        : EclBaseVanguard<TypeTag>(simulator), mpiRank()
```

Comment:
i think this is less obvious than an #else and being an int there is no cost.

Reply:
Even with MPI and valgrind (used without MPI bindings) might complain that mpiRank might be uninitialized. So the alternative with else is
```
    : EclBaseVanguard<TypeTag>(simulator)
{
#if HAVE_MPI
        mpiRank = 0
        MPI_Comm_rank(MPI_COMM_WORLD, &mpiRank);
#else 
        mpiRank=0
#endif
        this->callImplementationInit();
    }
```

Looks worse IMHO